analysis_driver/pipelines/demultiplexing.py

```diff
@@ -323,10 +323,6 @@ def _run(self):
                 # make sure the directory where the bam and qc files will go exists
                 makedirs(dirname(self.bam_path(run_element)), exist_ok=True)
                 makedirs(dirname(self.final_fastq_base(run_element)), exist_ok=True)
-                # get the reference genome
```

Comment:
Is there no need to get the reference genome at this stage anymore?

Reply:
`RunDataset` now implements this as a method. See the `self.dataset.reference_genome(run_element)` on line 329